miniapp/USERGUIDE.md

```diff
@@ -589,6 +589,26 @@ downloadedMiniApps.forEach {
 }
 ```
 
+### Passing parameters to a miniapp
+
+For a mini app, you can pass query parameters as String using `MiniApp.create` to be appended with miniapp's url.
+For example: `https://mscheme.1234/miniapp/index.html?param1=value1&param2=value2`
+
+```kotlin
+class MiniAppActivity : Activity(), CoroutineScope {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+    //...
+        launch {
+            val miniAppDisplay = withContext(Dispatchers.IO) {
+                MiniApp.instance().create("mini_app_id", miniAppMessageBridge, "param1=value1&param2=value2")
```

Comment:
Is this example correct? I think that this won't work because the third parameter is `MiniAppNavigator`.

Maybe it's better to use named parameters in the example:
```suggestion
                MiniApp.instance().create(
                    appId = "mini_app_id", 
                    miniAppMessageBridge = miniAppMessageBridge,
                    queryParams = "param1=value1&param2=value2"
                )
```

Reply:
Yeah, I'll update this part.